Engine/source/T3D/projectile.cpp

```diff
@@ -368,9 +367,8 @@ bool ProjectileData::preload(bool server, String &errorStr)
          if (Sim::findObject(decalId, decal) == false)
             Con::errorf(ConsoleLogEntry::General, "ProjectileData::preload: Invalid packet, bad datablockId(decal): %d", decalId);
 
-      String sfxErrorStr;
-      if( !sfxResolve( &sound, sfxErrorStr ) )
-         Con::errorf(ConsoleLogEntry::General, "ProjectileData::preload: Invalid packet: %s", sfxErrorStr.c_str());
+      if( !getProjectileSound() )
```

Comment:
Feel free to delete the erroneous/unnecessary spaces inside parenthesis (right of `(` and left of `)`). Optionally space between keyword and expression (except for operators, such as like `sizeof()`, casts)
example:
```cpp
if( !getProjectileSound() )
```
to
```cpp
if (!getProjectileSound())
```

Reply:
copied from the way it was done before